src/index.test.js

```diff
@@ -22,17 +22,20 @@ describe( 'RootComponent', () => {
 
 	it( 'renders without crashing', () => {
 		const app = renderer.create( <RootComponent /> );
+		renderer.act( () => {
+			jest.runAllTicks();
```

Comment:
Interesting!
Can we add a comment saying why/when is this needed? 🤔

Reply:
Added one [here](https://github.com/WordPress/gutenberg/pull/17228/files#diff-198fbd355f0b071c0a3bbb8a014e2ec9R36) for the same reason. It seems that when `EDIT_ENTITY_RECORD` is dispatched using [this generator function](https://github.com/WordPress/gutenberg/blob/master/packages/core-data/src/actions.js#L169) it's done so asynchronously, meaning the promise does not resolve immediately even during tests. This is related to the changes made on core data entities so I figured it was safe to update our tests for now. I'll try to find a good explanation for this and add something 👍